addons/stock/models/stock_picking.py

```diff
@@ -66,10 +66,11 @@ def _default_show_operations(self):
         help="If this checkbox is ticked, Odoo will automatically pre-fill the detailed "
         "operations with the corresponding products, locations and lot/serial numbers.")
     reservation_method = fields.Selection(
-        [('by_date', 'before scheduled date'), ('at_confirm', 'At Confirmation'), ('manual', 'Manually')],
+        [('at_confirm', 'At Confirmation'), ('manual', 'Manually'), ('by_date', 'Before scheduled date')],
```

Comment:
Seems strange to have `At Confirmation` and `Before scheduled date` :thinking: 
`At Confirmation`/`Before Scheduled Date` or `At confirmation`/`Before scheduled date` , no ?
It's a detail, but as you change this line precisely for this :sweat_smile: 
(Also, why change the order ? It makes a difference ?)

Reply:
It is to get the `Before scheduled date` at the end. The order impact the radio button order. And the `Before scheduled date` should be in the end to have a better UI for the form (`reservation_days_before` and `reservation_days_before_priority` closer).